modules/nw-udn-additional-config-details.adoc

```diff
@@ -8,26 +8,29 @@
 
 The following table explains additional configurations for `ClusterUserDefinedNetwork` and `UserDefinedNetwork` custom resources (CRs) that are optional. It is not recommended to set these fields without explicit need and understanding of OVN-Kubernetes network topology.
 
-.`UserDefinedNetworks` optional configurations
-[cols="1,1,7", options="header"]
+. Optional configurations for user-defined networks
+[cols="2,1,7", options="header"]
 |====
 
-|Field|Type|Description
+|CUDN field|UDN field|Type|Description
 
-|`spec.joinSubnets`
+|`spec.network.<topology>.joinSubnets`
+|`spec.<topology>.joinSubnets`
 |object
 |When omitted, the platform sets default values for the `joinSubnets` field of `100.65.0.0/16` for IPv4 and  `fd99::/64` for IPv6. If the default address values are used anywhere in the cluster's network you must override it by setting the `joinSubnets` field. If you choose to set this field, ensure it does not conflict with other subnets in the cluster such as the cluster subnet, the `default` network cluster subnet, and the masquerade subnet.
 
 The `joinSubnets` field configures the routing between different segments within a user-defined network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
 
-|`spec.ipam.lifecycle`
+|`spec.network.layer2.ipam.lifecycle`
+|`spec.layer2.ipam.lifecycle`
```

Comment:
`spec.network.layer2.ipam.lifecycle`
`spec.layer2.ipam.lifecycle`

Keep topology here as Or mentioned
`spec.network.<topology>.ipam.lifecycle`
`spec<topology>.ipam.lifecycle`

Instead add the details of topology  in the description column.

Reply:
It seems confusing to me to add the details of the topology in the description column given that the topology is covered in columns UDN and CUDN.